### ISSUE TITLE
Test failures if non-English locale is active

### ISSUE THREAD

**Issue by eserte** [CpanelInc/Overload-FileCheck#5]
With a German locale (LC_ALL=de_DE.UTF-8 or so):
```
# Failed test 'ERRNO set to "No such file or directory"'
# at t/mock-setting-errno.t line 33.
# +--------------------------------+----+--------------------------------+
# | GOT                            | OP | CHECK                          |
# +--------------------------------+----+--------------------------------+
# | Datei oder Verzeichnis nicht g | =~ | (?^:No such file or directory) |
# | efunden                        |    |                                |
# +--------------------------------+----+--------------------------------+

# Failed test 'ERRNO set to "Interrupted system call"'
# at t/mock-setting-errno.t line 113.
# +--------------------------------+----+--------------------------------+
# | GOT                            | OP | CHECK                          |
# +--------------------------------+----+--------------------------------+
# | Interrupt innerhalb eines Syst | =~ | (?^:Interrupted system call)   |
# | emaufrufs                      |    |                                |
# +--------------------------------+----+--------------------------------+
# Seeded srand with seed '20181024' from local date.
t/mock-setting-errno.t ................. 
Dubious, test returned 2 (wstat 512, 0x200)
Failed 2/19 subtests 
```

**Comment by toddr** [comment]
Probably should only be checking the $! + 0 not the string.

**Comment by eserte** [comment]
Additionally it has to be an older perl (<5.22) to reproduce the error.

**Comment by atoomic** [comment]
nice catch, yes only check the int value :-) should be good enough there

**Comment by atoomic** [comment]
would be fixed as part of release v0.003
